src/ui/modals/confirm_queue_clear.rs

```diff
@@ -120,7 +116,11 @@ impl Modal for ConfirmQueueClearModal<'_> {
                 }
                 _ => {}
             }
-        };
+        } else if key.code() == KeyCode::Tab {
```

Comment:
I like the idea, but not the implementation. I would like to minimize behavior not explicitly bound to a key. Maybe Next/PreviousTab could be used here? It is a bit obscure but those two binds should never do anything when a modal is open.

Reply:
Gosh, I didn't pay too much attention to the actions until now, thanks for the pointer! I'll change it.